src/map/GMap.js

```diff
@@ -238,6 +239,25 @@
         this._prevCenterLat = this.centerLat();
         this._prevCenterLong = this.centerLong();
         this._prevZoom = this.zoom();
+        
+        if (this.mapDrawingTools()) {
```

Comment:
This should be moved to the update method so that it can be enabled and disabled at runtime (anything in the "enter" method is only called once)?